spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -122,7 +122,7 @@ public Properties getConsumerProperties(String inputChannelName) {
 				channelConsumerProperties.setProperty(BinderPropertyKeys.CONCURRENCY,
 						Integer.toString(bindingProperties.getConcurrency()));
 			}
-			if (bindingProperties.isDurableSubscription() != null) {
+			if (Boolean.TRUE.equals(bindingProperties.isDurableSubscription())) {
```

Comment:
Since the default `durableSubscription` value is always `false` at the binder level, I thought one doesn't need to pass the value `durable=false` explicitly. no?